google/cloud/aiplatform/vizier/trial.py

```diff
@@ -109,6 +113,19 @@ def status(self) -> vz.TrialStatus:
         trial = self.api_client.get_trial(name=self.resource_name)
         return vz.TrialConverter.from_proto(trial).status
 
+    @property
+    def study(self) -> client_abc.StudyInterface:
+        """Returns the Study that this Trial belongs to."""
+        trial_path_components = self._parse_resource_name(self.resource_name)
+        return study.Study(
+            study.Study._format_resource_name(
+                project=trial_path_components["project"],
```

Comment:
I think if you're inheriting from `VertexAiResourceNounWithFutureManager`, you should be able to do `project=self.project` and `location=self.location`, since `project` and `location` are also properties in the base class.